src/index.js

```diff
@@ -49,9 +50,23 @@ body.addEventListener('click', (event) => {
       .then((response) => {
         const data = response.meals;
         body.appendChild(createCommentPop(mealId, data));
+        displayComments(mealId);
```

Comment:
- [ ] **I appreciate your background choice for your comment popup. However, it's breaking for the desktops. 
Please give it the appropriate width so that it covers the whole screen.**
![galti1](https://user-images.githubusercontent.com/104627775/187952889-e2135a4e-0106-4247-a59d-3012668d2311.PNG)

Reply:
Yes! Thanks for the observation! I just fixed it!